src/components/atoms/Button/button.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Button } from "react-bootstrap"
+
+const Button = (props) => {
+	return (
+		<Button>{props.buttonText}</Button>
```

Comment:
And also why do you need to create this as a separate component if this component just rendering `<Button>{props.buttonText}</Button>